content/en/blog/_posts/2023-12-11-node-expand-secret-moves-ga

```diff
@@ -0,0 +1,92 @@
+---
+layout: blog
+title: "Kubernetes 1.29: CSI Storage Resizing Authenticated and Generally Available in v1.29"
+date: 2023-12-11
+slug: csi-node-expand-secret-support-ga
+---
+**Authors:** Humble Chirammal, Louis Koo (deeproute.ai)
```

Comment:
Did you mean:
```markdown
**Authors:** Humble Chirammal (deeproute.ai), Louis Koo (deeproute.ai)
```
?

Reply:
no.. :). I have added the company name too.